devito/ops/operator.py

```diff
@@ -110,6 +110,14 @@ def _specialize_iet(self, iet, **kwargs):
     def hcode(self):
         return ''.join(str(kernel) for kernel in self._ops_kernels)
 
+    def _finalize(self, iet, parameters):
+        # Applies _finalize for each generated OPS kernel, which will generate the .h file
+        self._ops_kernels = [super(OperatorOPS, self)._finalize(
```

Comment:
`.extend([...])` instead of `= [ ...]` ?

Reply:
No. The kernels must be replaced. Otherwise, we will duplicate kernels, w/ and w/o declarations.

Reply:
I'm merging this, but I'm not a fan of methods -- even though they're private methods -- that replace mutable variables